loops.js

```diff
@@ -51,3 +68,13 @@
 // >> 9
 // ==========================================
 
+
+for (index = 0; index < 3; index += 1) {
+    console.log(index);
+}
+for (index = 3;index < 6; index +=1) {
+    console.log("> "+index);
+}
+for (index = 6;index < 10; index +=1) {
```

Comment:
Opdracht was "maak een loop", dit zijn 3 loops. Uitkomst is echter hetzelfde dus goed !